devel/create_distro_tarball.sh

```diff
@@ -0,0 +1,187 @@
+#!/bin/sh
+#
+# Copyright 2011 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: morlovich@google.com (Maksim Orlovich)
+#
+# The usual mechanism used to develop mod_pagespeed and build binaries is based
+# on merging all dependencies into a single source tree.  This script enables a
+# standard  untar/configure/make flow that does not bundle external libraries.
+# It generates the tarball including the configure (or rather generate.sh)
+# script.
+
+set -e  # exit script if any command returns an error
+set -u  # exit the script if any variable is uninitialized
+
+function usage {
+  echo -n "create_distro_tarball.sh [ --checkout_dir dir ] tarball"
+  echo " [ --git_tag tag | source_tree ]"
+  echo "examples of git_tag  would be 'master' or '1.11.33.4'"
```

Comment:
Can you clarify that `git_tag` can be a branch, tag, or commit id?  Basically, anything you can pass to `checkout -b`.